jwala-integration-test/src/test/resources/com/cerner/jwala/ui/selenium/configuration/uploadJvmResource.feature

```diff
@@ -0,0 +1,45 @@
+Feature: Upload Resource - Jvm Node
+
+  Scenario:Jvm Node
+    Given I logged in
+    And I am in the configuration tab
+    And I created a group with the name "seleniumGroup"
+    And I created a media with the following parameters:
+      | mediaName       | jdk1.8.0_92     |
+      | mediaType       | JDK             |
+      | archiveFilename | jdk1.8.0_92.zip |
+      | remoteDir       | d:/ctp          |
+    And I created a media with the following parameters:
+      | mediaName       | apache-tomcat-7.0.55     |
+      | mediaType       | Apache Tomcat            |
+      | archiveFilename | apache-tomcat-7.0.55.zip |
+      | remoteDir       | d:/ctp                   |
+    And I created a jvm with following parameters:
+      | jvmName    | seleniumJvm                |
+      | tomcat     | apache-tomcat-7.0.55 |
+      | jdk        | jdk1.8.0_92          |
+      | hostName   | localhost            |
+      | portNumber | 80                   |
+      | group      | seleniumGroup          |
+    And I am in the resource tab
+    And I expanded component "seleniumGroup"
+    And I expanded component "JVMs"
+    And I clicked on component "seleniumJvm"
+    And I clicked on add resource
+    And I fill in the "Deploy Name" field with "hello.xml"
+    And I fill in the "Deploy Path" field with "C:\ctp\app\instances"
+    And I choose the resource file "hello.xml.tpl"
+    And I click the upload resource dialog ok button
+    Then check resource uploaded successful
```

Comment:
We might be able to get away without a "when" but we should follow Cucumber scenario best practices and follow Given-When-Then for consistency